src/components/ec-table-head/ec-table-head.vue

```diff
@@ -15,7 +15,10 @@
         :colspan="column.span"
         scope="col"
       >
-        <span class="ec-table-head__cell-wrapper">
+        <span
+          class="ec-table-head__cell-wrapper"
+          :class="{'ec-table-head__cell-wrapper--justify-end': column.align === 'right'}"
```

Comment:
I don't think this is right. Table header should not align itself according to how the column content is aligned. When we have icons, they are aligned to the center but the header stays intact.

Reply:
I can check it with UX, because in the designs look like the header is align according to how the column content is aligned. Even in the columns which contain icons, the header text seems to be aligned to  center.
Maybe we have here a kind of misalignment between design and implementation.
![image](https://user-images.githubusercontent.com/4802418/79440431-47a5d280-7fd6-11ea-9bea-f74eb399c4b2.png)

Reply:
I see, let's wait what they say but we have to unify this, either fixing the icon type or removing right alignment.

I also see that the type of the column may be `currency`. Again, please check with designers that currencies should be always aligned to the right. Because then we should fix this on Trade finance tables too.

I think we should abandon the `--text-right`, `--text-center` and `--flex-end` modifiers and go with better names:
```css
__cell--is-type-icon {
  text-align: center;
}

__cell--is-type-currency {
  text-align: right
}
```

Reply:
After talk it with UX, it is added the alignment of the content of the columns for a new type, currency, and the header text of every column is aligned in the same way as its content.
Very well spotted!